DEVELOPMENT.md

```diff
@@ -0,0 +1,61 @@
+## Development
+
+This repository is a Yarn 4 monorepo containing the following packages:
+
+- [cypress](https://github.com/chromaui/test-archiver/tree/main/packages/cypress)
+  - Chromatic E2E Visual Test integration for Cypress
+- [playwright](https://github.com/chromaui/test-archiver/tree/main/packages/playwright)
+  - Chromatic E2E Visual Test integration for Playwright
+- [shared](https://github.com/chromaui/test-archiver/tree/main/packages/shared)
+  - Internal workspace package shared by the integrations above
+
+### Requirements
+
+- Node 18
+- Yarn 4
+
+### Getting Started
+
+- If you have `yarn 1` installed globally, it is recommended that you run `corepack enable` so that the version of yarn set in `packageManager` in `package.json` is used for this project
+- Run `yarn install`
+- Run `yarn build`
+
+### Testing
+
+Unit tests reside with the code being tested in the `src` directories.
+
+Each E2E integration also has its own suite of E2E tests using the framework it is targeting. These tests import directly from the integration in question and the results are sent to Chromatic to visually test that all is well with archiving and all the various pieces.
+
+These test suites use a basic Express server that's defined in `test-server` and can be run on its own using `yarn test:server`, but that is not necessary to run the tests.
+
+The E2E test cases for each integration should match each other as much as possible. Changes or additions made to one should be matched in the others.
+
+#### Running the Tests
+
+- Unit tests: `yarn test:unit`
+- Playwright: `yarn test:playwright`, then `yarn archive-storybook:playwright` to see the archived UI
+- Cypress: `yarn test:cypress`, then `yarn archive-storybook:cypress` to see the archived UI
```

Comment:
Do we no longer have to run `yarn build` before running `yarn test:*`? I've been running the two together like that to make sure package changes make their way into the tests (even if the shared package is what changed).

Reply:
Hm good callout. We do have to run the build. I can make that explicit in the docs. I haven't looked into adding something like `build:watch` to watch for changes, which could make this less of a pain.

Reply:
Yeah. In the short-term, we could just have the `test:x` commands be `yarn build && yarn test:x`, what do you think?

Reply:
Yeah good question, I was wondering about that too. That might be better, but could be slightly annoying when it's not necessary (like when only adding/changing the test suite) 🤔

Reply:
I'll update the docs now, and we can change that in a follow up.

Reply:
Added a note!